### ISSUE TITLE
dod.tideGauge() fails

### ISSUE THREAD

**Issue by dankelley** [dankelley/dod#12]
```R
> library(dod)
> ofile <- dod.tideGauge(491)
Error in s$code : $ operator is invalid for atomic vectors
> library(oce)
Loading required package: gsw
> ofile <- dod.tideGauge(491)
Error in s$code : $ operator is invalid for atomic vectors
> traceback()
5: FUN(X[[i]], ...)
4: lapply(X = X, FUN = FUN, ...)
3: sapply(d, function(s) identical(ID, as.integer(s$code))) at tide_gauge.R#207
2: which(sapply(d, function(s) identical(ID, as.integer(s$code)))) at tide_gauge.R#207
1: dod.tideGauge(491)
```

**Comment by dankelley** [comment]
It looks as though they have changed the API.  I'll look into this.

**Comment by dankelley** [comment]
I've got the initial steps done.  Just for my notes, I now need to grab just one of the entries. I imagine I need the first ID, although frankly the word 'phenomenon' would have made me think I needed the second ID.
```
Browse[1]> d$timeSeries[w]
[[1]]
                        id     code
1 5cebf1e23d0f4a073c4bbfa8      wlp
2 5cebf1e23d0f4a073c4bbfa9      wlf
3 5cebf1e23d0f4a073c4bbfaa      wlo
4 5d9dd7cf33a9f593161c4055 wlp-hilo
                                      nameEn
1                    Water level predictions
2 Water level forecasts generated by the FMS
3                 Water level official value
4              High and Low Tide Predictions
                                           nameFr             phenomenonId
1                    Prédictions de niveaux d'eau 5ce598df487b84486892821c
2 Prévisions de niveaux d'eau générées par le FMS 5ce598df487b84486892821c
3                 Niveau d'eau, valeur officielle 5ce598df487b84486892821c
4           Prédictions de pleines et basses mers 5ce598df487b84486892821c
    owner
1 CHS-SHC
2 CHS-SHC
3 CHS-SHC
4 CHS-SHC
```

**Comment by dankelley** [comment]
I'll walk through this using the API manually, starting with the ID I know to be 5cebf1e23d0f4a073c4bbfac from code I've already written

```
curl -X 'GET' \
  'https://api.iwls-sine.azure.cloud-nuage.dfo-mpo.gc.ca/api/v1/stations/5cebf1e23d0f4a073c4bbfac/metadata' \
  -H 'accept: application/json'
```

yields

```
{
  "id": "5cebf1e23d0f4a073c4bbfac",
  "code": "00491",
  "officialName": "Bedford Institute",
  "latitude": 44.682262,
  "longitude": -63.613239,
  "type": "PERMANENT",
  "status": "OK",
  "operating": true,
  "expectedProductivityPerHour": 60,
  "owner": "CHS-SHC",
  "chsRegionCode": "ATL",
  "provinceCode": "NS",
  "classCode": "A",
  "isTidal": true,
  "timeZoneCode": "Canada/Atlantic",
  "tideTableId": "5da0907154c1370c6037fcc6",
  "isTideTableReferencePort": false,
  "referencePortStationId": "5cebf1df3d0f4a073c4bbcbb",
  "timeSeries": [
    {
      "id": "5cebf1e23d0f4a073c4bbfa8",
      "code": "wlp",
      "nameEn": "Water level predictions",
      "nameFr": "Prédictions de niveaux d'eau",
      "phenomenonId": "5ce598df487b84486892821c",
      "owner": "CHS-SHC"
    },
    {
      "id": "5cebf1e23d0f4a073c4bbfa9",
      "code": "wlf",
      "nameEn": "Water level forecasts generated by the FMS",
      "nameFr": "Prévisions de niveaux d'eau générées par le FMS",
      "phenomenonId": "5ce598df487b84486892821c",
      "owner": "CHS-SHC"
    },
    {
      "id": "5cebf1e23d0f4a073c4bbfaa",
      "code": "wlo",
      "nameEn": "Water level official value",
      "nameFr": "Niveau d'eau, valeur officielle",
      "phenomenonId": "5ce598df487b84486892821c",
      "owner": "CHS-SHC"
    },
    {
      "id": "5d9dd7cf33a9f593161c4055",
      "code": "wlp-hilo",
      "nameEn": "High and Low Tide Predictions",
      "nameFr": "Prédictions de pleines et basses mers",
      "phenomenonId": "5ce598df487b84486892821c",
      "owner": "CHS-SHC"
    }
  ],
  "datums": [
    {
      "code": "CGVD28",
      "offset": -0.81,
      "offsetPrecision": 2
    },
    {
      "code": "NAD83_CSRS",
      "offset": -21.68,
      "offsetPrecision": 2
    }
  ],
  "heights": [
    {
      "heightTypeId": "5cec2eba3d0f4a04cc64d5d7",
      "value": 2.21,
      "valuePrecision": 2
    },
    {
      "heightTypeId": "5cec2eba3d0f4a04cc64d5d8",
      "value": -0.05,
      "valuePrecision": 2
    },
    {
      "heightTypeId": "5cec2eba3d0f4a04cc64d5ce",
      "value": 2.17,
      "valuePrecision": 2
    },
    {
      "heightTypeId": "5cec2eba3d0f4a04cc64d5d3",
      "value": -0.01,
      "valuePrecision": 2
    },
    {
      "heightTypeId": "5cec2eba3d0f4a04cc64d5d4",
      "value": 1.83,
      "valuePrecision": 2
    },
    {
      "heightTypeId": "5cec2eba3d0f4a04cc64d5d5",
      "value": 0.36,
      "valuePrecision": 2
    },
    {
      "heightTypeId": "5cec2eba3d0f4a04cc64d5cf",
      "value": 1.77,
      "valuePrecision": 2
    },
    {
      "heightTypeId": "5cec2eba3d0f4a04cc64d5d0",
      "value": 0.46,
      "valuePrecision": 2
    },
    {
      "heightTypeId": "5cec2eba3d0f4a04cc64d5d2",
      "value": 1.1,
      "valuePrecision": 2
    },
    {
      "heightTypeId": "5cec2eba3d0f4a04cc64d5d1",
      "value": 2.95,
      "valuePrecision": 2,
      "date": "1976-03-17"
    },
    {
      "heightTypeId": "5cec2eba3d0f4a04cc64d5d6",
      "value": -0.32,
      "valuePrecision": 2,
      "date": "2007-03-21"
    }
  ]
}
```

**Comment by dankelley** [comment]
This is solved in "main" commit 2b6548dfa2b2e33ecfea22887b3f6b7e362bdca2.  I bumped the version number.

I see that it won't work with `pkgdown::build_site()`.  I'll get to that in a few days, likely not before the weekend.  This is likely a problem with setting up a temporary directory, etc.
